runtime/centrifuge/src/lib.rs

```diff
@@ -418,6 +428,93 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	// To be used if we want to register a particular asset in the chain spec, when running the chain locally.
+	pub LiquidityPoolsPalletIndex: PalletIndex = <LiquidityPools as PalletInfoAccess>::index() as u8;
+}
+
+impl pallet_liquidity_pools::Config for Runtime {
+	type AccountConverter = AccountConverter<Runtime>;
+	type AdminOrigin = EnsureRootOr<TwoThirdOfCouncil>;
+	type AssetRegistry = OrmlAssetRegistry;
+	type Balance = Balance;
+	type CurrencyId = CurrencyId;
+	type ForeignInvestment = Investments;
+	type GeneralCurrencyPrefix = cfg_primitives::liquidity_pools::GeneralCurrencyPrefix;
+	type OutboundQueue = WrapperOutboundQueue;
+	type Permission = Permissions;
+	type PoolId = PoolId;
+	type PoolInspect = PoolSystem;
+	type Rate = Rate;
+	type RuntimeEvent = RuntimeEvent;
+	type Time = Timestamp;
+	type Tokens = Tokens;
+	type TrancheCurrency = TrancheCurrency;
+	type TrancheId = TrancheId;
+	type TrancheTokenPrice = PoolSystem;
+	type TreasuryAccount = TreasuryAccount;
+	type WeightInfo = ();
+}
+
+type LiquidityPoolsMessage =
+	pallet_liquidity_pools::Message<Domain, PoolId, TrancheId, Balance, Rate>;
+
+/// The WrapperOutboundQueue serves as a filter for outbound LP messages that we
+/// want to allow initially.
+pub struct WrapperOutboundQueue;
+
+impl OutboundQueue for WrapperOutboundQueue {
+	type Destination = Domain;
+	type Message = LiquidityPoolsMessage;
+	type Sender = AccountId;
+
+	fn submit(
+		sender: Self::Sender,
+		destination: Self::Destination,
+		msg: Self::Message,
+	) -> DispatchResult {
+		match msg {
+			LiquidityPoolsMessage::AddCurrency { .. }
+			| LiquidityPoolsMessage::UpdateMember { .. }
+			| LiquidityPoolsMessage::AddPool { .. }
+			| LiquidityPoolsMessage::AddTranche { .. }
+			| LiquidityPoolsMessage::UpdateTrancheTokenPrice { .. } => {
+				<LiquidityPoolsGateway as OutboundQueue>::submit(sender, destination, msg)
+			}
+			_ => Err(DispatchError::Other("unsupported outbound message")),
+		}
+	}
+}
+
+parameter_types! {
+	pub const MaxIncomingMessageSize: u32 = 1024;
+}
+
+impl pallet_liquidity_pools_gateway::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<TwoThirdOfCouncil>;
+	type InboundQueue = DummyInboundQueue;
+	type LocalEVMOrigin = pallet_liquidity_pools_gateway::EnsureLocal;
+	type MaxIncomingMessageSize = MaxIncomingMessageSize;
+	type Message = LiquidityPoolsMessage;
+	type Router = liquidity_pools_gateway_routers::DomainRouter<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeOrigin = RuntimeOrigin;
+	type WeightInfo = ();
+}
+
+/// DummyInboundQueue will be used in the first phase of testing in order to
+/// ensure that no incoming messages will be processed.
+pub struct DummyInboundQueue;
```

Comment:
Why would we want this on Centrifuge?

Reply:
We do want to check the gatway, but do not want to hit the connectors logic yet.

Reply:
FI is not in yet.

Reply:
Ah ok I see.